meson.build

```diff
@@ -16,6 +16,8 @@ graphene_micro_version = version_array[2].to_int()
 
 graphene_api_version = '@0@.0'.format(graphene_major_version)
 
+graphene_string = '@0@-@1@'.format(meson.project_name(), graphene_api_version)
```

Comment:
One last option, `graphene_api_path`, as it's true that it's mostly used for paths and file names, but it's also related to the api version. What's your opinion?